lib/time/time.Time.js

```diff
@@ -361,7 +361,13 @@
 	};
 
 	function pad( number, digits ) {
-		return ( 1e12 + Math.abs( number ) + '' ).slice( -digits );
+		number = '' + Math.abs( number );
```

Comment:
I don't think of `String` as a constructor since nobody ever uses it that way. On the other hand, it clearly states what you are trying to do.